GitVersion .Net Core (Preview) Azure DevOps task error

Just tried out the new Azure DevOps task for .Net Core. It fails on a build with the following message

```
##[error]Unhandled: Cannot find module 'argparse'
```

It still fails for me in the same way although I'm not entirely confident that Azure DevOps is running the latest version of the package. I tried uninstalling the build task and installing it again. Even running with debug logging on the build doesn't actually provide the package version. It just says 5.0.0.

This is the log for the Initialize Job step
```
##[section]Starting: Initialize job
Current agent version: '2.147.1'
##[debug]Primary repository: Bankless. repository type: Git
Prepare build directory.
##[debug]Creating build directory: 'D:\a\1'
##[debug]Delete existing artifacts directory: 'D:\a\1\a'
##[debug]Creating artifacts directory: 'D:\a\1\a'
##[debug]Delete existing test results directory: 'D:\a\1\TestResults'
##[debug]Creating test results directory: 'D:\a\1\TestResults'
##[debug]Creating binaries directory: 'D:\a\1\b'
##[debug]Creating source directory: 'D:\a\1\s'
Set build variables.
Download all required tasks.
Downloading task: GitVersionNetCore
##[debug]Task 'GitVersionNetCore' has been downloaded into 'D:\a\_tasks\GitVersionNetCore_edf331e1-d1c0-413a-9735-fce0b22a46f5\5.0.0'.
Downloading task: DotNetCoreCLI
##[debug]Task 'DotNetCoreCLI' has been downloaded into 'D:\a\_tasks\DotNetCoreCLI_5541a522-603c-47ad-91fc-a4b1d163081b\2.147.2'.
Downloading task: PublishBuildArtifacts
##[debug]Task 'PublishBuildArtifacts' has been downloaded into 'D:\a\_tasks\PublishBuildArtifacts_2ff763a7-ce83-4e1f-bc89-0ae63477cebe\1.142.2'.
##[debug]Task 'GitVersion' has following condition: 'succeeded()'.
##[debug]Parsing expression: <succeeded()>
##[debug]succeeded
##[debug](
##[debug])
##[debug]Task 'Restore' has following condition: 'succeeded()'.
##[debug]Parsing expression: <succeeded()>
##[debug]succeeded
##[debug](
##[debug])
##[debug]Task 'Build' has following condition: 'succeeded()'.
##[debug]Parsing expression: <succeeded()>
##[debug]succeeded
##[debug](
##[debug])
##[debug]Task 'Test' has following condition: 'succeeded()'.
##[debug]Parsing expression: <succeeded()>
##[debug]succeeded
##[debug](
##[debug])
##[debug]Task 'Publish' has following condition: 'succeeded()'.
##[debug]Parsing expression: <succeeded()>
##[debug]succeeded
##[debug](
##[debug])
##[debug]Task 'Publish Artifact' has following condition: 'succeeded()'.
##[debug]Parsing expression: <succeeded()>
##[debug]succeeded
##[debug](
##[debug])
Start tracking orphan processes.
##[section]Finishing: Initialize job
```

The GitVersion step then has this
```
##[debug]Evaluating condition for step: 'GitVersion'
##[debug]Evaluating: succeeded()
##[debug]Evaluating succeeded:
##[debug]=> True
##[debug]Result: True
##[section]Starting: GitVersion
==============================================================================
Task         : GitVersion .NET Core Task
Description  : Easy Semantic Versioning (http://semver.org) for projects using Git
Version      : 5.0.0
Author       : GitVersion Contributors
Help         : See the [documentation](http://gitversion.readthedocs.org/en/latest/) for help
==============================================================================
##[debug]agent.TempDirectory=D:\a\_temp
##[debug]loading inputs and endpoints
##[debug]loading ENDPOINT_AUTH_PARAMETER_SYSTEMVSSCONNECTION_ACCESSTOKEN
##[debug]loading ENDPOINT_AUTH_SCHEME_SYSTEMVSSCONNECTION
##[debug]loading ENDPOINT_AUTH_SYSTEMVSSCONNECTION
##[debug]loading INPUT_UPDATEASSEMBLYINFO
##[debug]loaded 4
##[debug]Agent.ProxyUrl=undefined
##[debug]Agent.CAInfo=undefined
##[debug]Agent.ClientCert=undefined
##[debug]Agent.SkipCertValidation=undefined
##[debug]task result: Failed
##[error]Unhandled: Cannot find module 'argparse'
##[debug]Processed: ##vso[task.issue type=error;]Unhandled: Cannot find module 'argparse'
##[debug]Processed: ##vso[task.complete result=Failed;]Unhandled: Cannot find module 'argparse'
##[section]Finishing: GitVersion
```

--- a/src/GitVersionTfsTask/package.json
+++ b/src/GitVersionTfsTask/package.json
@@ -15,14 +15,14 @@
     "license": "MIT",
     "dependencies": {
         "q": "1.5.1",
-        "azure-pipelines-task-lib": "2.7.7"
+        "azure-pipelines-task-lib": "2.7.7",
+        "argparse": "^1.0.10"
     },
     "devDependencies": {
         "@types/argparse": "^1.0.36",
         "@types/minimatch": "3.0.3",
         "@types/node": "11.9.5",
         "@types/q": "1.5.1",
-        "argparse": "^1.0.10",
         "copy-node-modules": "^1.1.1",
         "tfx-cli": "^0.6.4",
         "typescript": "^3.3.3333"

